app/src/components/googleMap.tsx

```diff
@@ -33,7 +36,10 @@ interface MapState {
     destinationsRequestModel: FlightDestinationRequest;
     isLoading?: boolean;
     error?: string;
-    selectedAirportlabel: string;
+    selectedAirportlabel: string;  //label and Id is not the same thing
+    departureAirportId: string;
+    departureLat: number;
```

Comment:
why don't you use object? like:
departureCoords:Coordinates

Reply:
and this